gitbook-docs/market/cmd-reference/statement.md

```diff
@@ -1,75 +1,104 @@
 ---
-description: Scripts to manage statements/circuits
+description: Managing circuits and statements
 ---
 
-# statement
+# Statement utilities
 
+This page describes scripts for circuit developers and proof producers,
+`prepare_statement.py` and `statement_tools.py`.
 
+Make sure you've [signed up](user.md) before continuing.
 
-### Prepare Statement
+## Preparing circuit as a statement
 
-Statements/circuits generated by the zkLLVM compiler need to have additional metadata attached to them before they can be published on the proof market.
+Circuits are stored as a *statement* structure on the Proof Market.
```

Comment:
The structure is something internal. Okay, it's some data structure inside. But to operate it, we don't need to know it.


```suggestion
Circuits are stored in the form of statements on the Proof Market.
```

Reply:
Thanks, fixed